apis/v1alpha1/annotations.go

```diff
@@ -0,0 +1,7 @@
+package v1alpha1
+
+import "fmt"
+
+var (
+	ResourceIDAnnotation = fmt.Sprintf("%s/resource-id", GroupVersion.Group)
```

Comment:
Not sure I understand why what this annotation is used for - can you please expand more on this?

Reply:
I have added a comment.